src/plugins/auto_batch/tests/functional/behavior/ov_plugin/auto_batching_tests.cpp

```diff
@@ -31,6 +31,42 @@ INSTANTIATE_TEST_SUITE_P(smoke_AutoBatching_test,
                                             ::testing::ValuesIn(num_batch)),
                          AutoBatching_Test_DetectionOutput::getTestCaseName);
 
+INSTANTIATE_TEST_SUITE_P(nightly_AutoBatching_CPU,
```

Comment:
```suggestion
INSTANTIATE_TEST_SUITE_P(nightly_CPU_AutoBatching,
```
Let's put CPU/GPU before the BATCH name, so that we can filter nightly_CPU_* for CPU nightly environment.